userspace/libscap/scap_savefile.c

```diff
@@ -1165,6 +1170,28 @@ static int32_t scap_read_proclist(scap_t *handle, gzFile f, uint32_t block_lengt
 		totreadsize += readsize;
 
 		//
+		// exepath
+		//
+		readsize = gzread(f, &(stlen), sizeof(uint16_t));

```

Comment:
I like this. However, doing so will break the compatibility with older trace files, because sysdig will now expect every trace file to contain the exepath:

```
$ sudo ./sysdig -r ~/trace.scap 
Could not initialize reader: invalid rootlen 4806
```

You should probably introduce a new block type after `PL_BLOCK_TYPE_V6`, if you search all the usages in the code it should be clear how to bump it.